python/src/rest_api/aac_rest_app.py

```diff
@@ -0,0 +1,531 @@
+from fastapi import FastAPI, HTTPException, BackgroundTasks, responses, exceptions, Request
+from http import HTTPStatus
+import logging
+from typing import Optional
+import os
+
+from aac.context.definition_parser import DefinitionParser
+from aac.context.language_context import LanguageContext
+from aac.context.language_error import LanguageError
+from aac.context.definition import Definition
+from aac.execute.plugin_runner import AacCommand
+from aac.execute.plugin_manager import get_plugin_manager
+from aac.in_out.files.aac_file import AaCFile
+from aac.in_out.files.find import find_aac_files, is_aac_file
+from aac.in_out.paths import sanitize_filesystem_path
+from aac.in_out.parser import parse, ParserError
+from aac.execute.aac_execution_result import ExecutionStatus
+
+from rest_api.models.command_model import (
+    CommandModel,
+    CommandRequestModel,
+    CommandResponseModel,
+    to_command_model,
+)
+from rest_api.models.definition_model import DefinitionModel, to_definition_class, to_definition_model
+from rest_api.models.file_model import FileModel, FilePathModel, FilePathRenameModel, to_file_model
+
+app = FastAPI()
+
+AVAILABLE_AAC_FILES: list[AaCFile] = []
+WORKSPACE_DIR: str = os.getcwd()
+
+# File CRUD Operations
+
+def _get_files_in_context() -> list[AaCFile]:
+    """
+    Returns a list of all files contributing definitions to the active context.
+
+    Args:
+        active_context (LanguageContext):  The active language context.
+
+    Returns:
+        A list of all files contributing definitions to the active context.
+    """
+    return list({definition.source for definition in active_context.definitions})
+
+def _get_file_in_context_by_uri(uri: str) -> Optional[AaCFile]:
+    """
+    Return the AaCFile object by uri from the context or None if the file isn't in the context.
+
+    Args:
+        active_context (LanguageContext):  The active language context.
+        uri (str): The string uri to search for.
+
+    Returns:
+        An optional AaCFile if it's present in the context, otherwise None.
+    """
+    for definition in active_context.definitions:
+        if definition.source.uri == uri:
+            return definition.source
+
+def _get_definitions_by_file_uri(file_uri: str) -> list[Definition]:
+    """
+    Return a subset of definitions that are sourced from the target file URI.
+
+    Args:
+        active_context (LanguageContext):  The active language context.
```

Comment:
```suggestion
```

Doesn't seem like it is used within the logic of this method, so should be good to remove.